remote-api/src/test/java/org/alfresco/rest/api/impl/CategoriesImplTest.java

```diff
@@ -93,6 +102,8 @@ public void shouldNotGetRootCategoryById()
         then(nodesMock).shouldHaveNoMoreInteractions();
         then(nodeServiceMock).should().getParentAssocs(categoryRootNodeRef);
         then(nodeServiceMock).shouldHaveNoMoreInteractions();
+        then(categoryServiceMock).shouldHaveNoInteractions();
+        then(authorityServiceMock).shouldHaveNoMoreInteractions();
```

Comment:
Shouldn't be used here `.shouldHaveNoInteractions()` instead `.shouldHaveNoMoreInteractions()` (without "More")? Here and in bellow test classes related to `get()`?

Reply:
Good point - most likely a copy-paste mistake here. Fixing that.